src/web/components/elements/BlockquoteBlockComponent.stories.tsx

```diff
@@ -30,6 +30,11 @@ export const Unquoted = () => {
 					display: Display.Standard,
 					theme: Pillar.News,
 				})}
+				format={{
+					design: Design.Article,
+					display: Display.Standard,
+					theme: Pillar.News,
+				}}
```

Comment:
Would be nice if there was a way we could do this without having to duplicate the `palette` prop - but it's not really a huge deal since it's just for stories

Reply:
the difference here is subtle, you can see `palette` uses `decidePalette`. But granted this looks like a lot of unnecessary repetition, although for stories I think that's ok

Reply:
Probably in another PR, but it's worth removing `palette` as a prop and just using `format` to `decidePalette` at the usage point rather than drilling it all the way down.